examples/demo-app/webpack.config.js

```diff
@@ -69,7 +69,11 @@ const CONFIG = {
       'MapboxAccessToken',
       'DropboxClientId',
       'MapboxExportToken',
-      'CartoClientId'
+      'CartoClientId',
+      'FSQStudioClientId',
```

Comment:
Again recommend just going with `Foursquare`.
```suggestion
      'FoursquareClientId',
```

Reply:
yeah, already replaced in the first PR